### PR TITLE
Master

### PR DESCRIPTION
**Pull request by b3m2a1**
Moved file out of downloads per
https://blog.github.com/2012-12-12-goodbye-uploads/

Added paclet support

### CODE REVIEW

File: Stylesheets/pastelSolarized.nb [hunk 1]
```diff
@@ -0,0 +1,287 @@
+(* Content-type: application/vnd.wolfram.mathematica *)
```
Comment (halirutan) [inline Stylesheets/pastelSolarized.nb]: While we are at it, should we clear the notebooks from the Cache, Window position, etc.. entries?

File: Stylesheets/pastelSolarized.nb [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+ FontFamily->"Optima"],
+
+Cell[StyleData["Text"],
+ TextJustification->1,
+ FontFamily->"Optima",
+ FontSize->12]
+}, Open  ]]
+},
+WindowSize->{740, 756},
+WindowMargins->{{Automatic, 64}, {Automatic, 0}},
```
Comment (halirutan) [inline Stylesheets/pastelSolarized.nb]: I had bad experiences with these settings. While here it looks OK, it was possible (I use 2 screens) at least in earlier versions that a notebook was off the screen when opening it.

File: Stylesheets/pastelSolarized.nb [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+FrontEndVersion->"8.0 for Mac OS X x86 (32-bit, 64-bit Kernel) (October 5, \
+2011)",
+StyleDefinitions->"PrivateStylesheetFormatting.nb"
+]
+(* End of Notebook Content *)
```
Comment (halirutan) [inline Stylesheets/pastelSolarized.nb]: Maybe remove the cache info

**Review by halirutan — COMMENTED** [review]
Should we clean the notebooks first?

**Comment by b3m2a1** [comment]
I expunged all of that via the appropriate options. I can also clear those options by hand if you think it worth it.

**Review by halirutan — APPROVED** [review]

**Pull request merged**